tsdb/engine/tsm1/cache.go

```diff
@@ -231,10 +231,10 @@ func (c *Cache) WriteMulti(values map[string][]Value) error {
 	}
 	c.mu.RUnlock()
 
+	c.mu.Lock()
```

Comment:
benchmarks (200000 keys , each key include 4000 values)：

```
before:

Sosaras-MacBook-Air:tsm1 Sosara$ go test -v -test.run TestCache_Bench_CacheWriteMulti
=== RUN   TestCache_Bench_CacheWriteMulti
--- PASS: TestCache_Bench_CacheWriteMulti (9.01s)

after:

Sosaras-MacBook-Air:tsm1 Sosara$ go test -v -test.run TestCache_Bench_CacheWriteMulti
=== RUN   TestCache_Bench_CacheWriteMulti
--- PASS: TestCache_Bench_CacheWriteMulti (8.89s)

```

test many times in the same environment.

and the `c.entry(k).add(v)` not only requires an `RLock` , but also a `Lock`

@e-dard